python/lsst/ap/association/loadDiaCatalogs.py

```diff
@@ -93,24 +134,21 @@ def run(self, exposure, apdb, doLoadForcedSources=True):
         RuntimeError
             Raised if the Database query failed to load DiaObjects.
         """
-        region = self._getRegion(exposure)
-        schema = readSchemaFromApdb(apdb)
+        region = regionTime.region
+        schema = readSchemaFromApdb(self.apdb)
 
         # This is the first database query.
-        try:
-            diaObjects = self.loadDiaObjects(region, apdb, schema)
-        except (OperationalError, ProgrammingError) as e:
-            raise RuntimeError(
-                "Database query failed to load DiaObjects; did you call "
-                "make_apdb.py first? If you did, some other error occurred "
-                "during database access of the DiaObject table.") from e
+        diaObjects = self.loadDiaObjects(region, schema)
 
-        dateTime = exposure.visitInfo.date
+        # Load diaSources and forced sources up to the time of the exposure
+        # The timespan includes significant padding, so use the midpoint to
```

Comment:
"may include"? I do hope we can cut back once the Summit gets us better metadata, and it's a non-issue when running in batch.